plasmapy/mathematics/mathematics.py

```diff
@@ -197,6 +197,10 @@ def Fermi_integral(x, j):
 
     .. math::
         F_j (x) = -Li_{j+1}\left(-e^{x}\right)
+        
+    Warning: at present this function is limited to relatively small
+    arguments due to limitations in the mpmath package's implementation
+    of polylog.
```

Comment:
Could probably do with something along the lines of:

```
try:
    value = polylog(...)
except NonImplementedError:
    raise(NonImplementedError(this_very_string))
```

Reply:
Added to the to do list.